jax/experimental/jax2tf/tests/tf_test_util.py

```diff
@@ -23,22 +23,16 @@
 from jax.config import config
 from jax import dtypes
 from jax.experimental import jax2tf
+from jax.experimental.jax2tf.tests import correctness_stats as jcs
```

Comment:
I think that the Google style rule would prefer:
```
from jax.experimental.jax2tf.tests import correctness_stats
```

We do not strictly use the Google style rules in this project, but I think in this case there is little benefit from the more cryptic `jcs` to be worth it (there are few uses).

Reply:
Addressed in `697c06d`